gcp_variant_transforms/testing/integration/run_preprocessor_tests.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""Integration testing runner for Variant Transforms' Preprocessor pipeline.
+
+You may run this test in any project (the test files are publicly accessible).
+Execute the following command from the root source directory:
+python gcp_variant_transforms/testing/integration/run_preprocessor_tests.py \
+  --project myproject \
+  --staging_location gs://mybucket/staging \
+  --temp_location gs://mybucket/temp \
+  --logging_location gs://mybucket/temp/integration_test_logs
+
+To keep the reports that this test creates, use the --keep_reports option.
+
+It runs all integration tests inside
+`gcp_variant_transforms/testing/integration/preprocessor_tests`.
+"""
+
+import argparse
+import multiprocessing
+import sys
+import os
+from typing import Dict, List  # pylint: disable=unused-import
+
+from apache_beam.io import filesystems
```

Comment:
Question: Do you really want this to be file system agnostic or are you okay with using GCS libraries directly? The reason that I am asking is that this is the only dependency of integration test modules on `apache_beam` (and if you think about it, it is expected that these modules do not depend on Beam because we are not running Dataflow pipelines directly here). The only benefit that I can see is the file system agnosticism baked into `apache_beam.io`.